### ISSUE TITLE
DBTag: removable button with a height of only 22px

### ISSUE THREAD

**Issue by mfranzke** [db-ui/mono#2793]
### Which generators are impacted?

- [X] All
- [ ] HTML
- [ ] React
- [ ] Angular
- [ ] Vue
- [ ] Web components
- [ ] Power Apps

### Reproduction case

The clickable area of the removable button is only 22px height, compare to https://db-ui.github.io/mono/review/main/components/data-display/tag/overview?page=behaviour

### Expected Behaviour

The clickable area of the removable button should be 24px height.

### Browser version

None

### Add any other context about the problem here.

We might increase it e.g. by a pseudo content element that extends to 24px.
